scripts/setup-antigravity.py

```
@@ -0,0 +1,167 @@
+
+def install():
+    ag_dir = get_antigravity_dir()
+    ag_dir.mkdir(parents=True, exist_ok=True)
+    settings_file = ag_dir / "settings.json"
+    
+    data = {}
+    if settings_file.exists():
+        try:
+            with open(settings_file, "r", encoding="utf-8") as f:
+                data = json.load(f)
+        except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)